presto-spi/src/main/java/io/prestosql/spi/StandardErrorCode.java

```diff
@@ -112,6 +112,7 @@
     NESTED_RECURSIVE(89, USER_ERROR),
     INVALID_LIMIT_CLAUSE(90, USER_ERROR),
     INVALID_ORDER_BY(91, USER_ERROR),
+    UPDATE_COLUMN_DOESNT_EXIST(92, USER_ERROR),
```

Comment:
We can use the existing `COLUMN_NOT_FOUND`

Reply:
Changed.